utilities/production_params.R

```diff
@@ -111,7 +130,28 @@ county_corrections_params  <- zookeeper::default_county_params(
   signal = county_forecaster_signals$signal[1] # only correct cases
 )
 
-county_corrector  <- zookeeper::make_zyzzyva_corrector(params = county_corrections_params)
+county_corrector  <- zookeeper::make_zyzzyva_corrector(
+  params = county_corrections_params,
+  manual_flags = tibble(
+    data_source = "jhu-csse",
+    signal = "confirmed_incidence_num",
+    geo_value = c(
+      ## from JHU-CSSE notes 2021-04-17, 2021-04-18
+      "29077", "29095", "29183", "29189",
+      "01097"
+    ),
+    time_value = list(
```

Comment:
1. Again `lubridate::ymd()`. 
2. (aside) Since this is just one date per location, it doesn't need the list (though it's fine). You can do:
```r
time_value = lubridate::ymd(c("2020-04-17", ... , "2021-04-13"))
```

Reply:
1. Fixed.
2. Just kept as list for now because it was easy and highlights the fact that it could be a list for later.